.travis.yml

```diff
@@ -1,7 +1,10 @@
 language: go
-go: 1.2
+go:
+    - 1.6.3
+before_install:
+    - gotools=golang.org/x/tools
 install:
-    - go get -v code.google.com/p/go.tools/cmd/cover
+    - go get -v $gotools/cmd/cover
```

Comment:
Why is the `gotools` variable needed, instead of just doing `go get -v golang.org/x/tools/cmd/cover` here?